ownership.md

```diff
@@ -0,0 +1,54 @@
+
+On top of that, every codebase should have ongoing security and upkeep maintenance applied to it. *This may only be lightweight and relatively infrequent when the project is low priority/risk.*
+
+## Baseline requirements for production services and applications
+
+N.B. This guidance only intended as a minimum baseline; in practice the expectation will be much higher for critical projects.
+
+### CI/CD
+- All source code should be version controlled using [GitHub](./github.md)
+- CI/CD should be employed
+    - An appropriate testing strategy should be considered. We do not aim for a specific % of test coverage, but important business logic should be unit tested
+    - For CI, we use GitHub Actions and [Teamcity](https://teamcity.gutools.co.uk/)
+    - For most (non-library) projects, deployment will be done via [Riff-Raff](https://riffraff.gutools.co.uk/)
+
+### Security
+- A basic [security](./security.md) assessment should be performed to understand the risks and available controls. E.g. 
+authentication, network security, encryption, secret management. Expert guidance from outside the team should sought for high risk applications (e.g. processing user data)
+- Any dependency manifest files should be scanned using [Snyk Open Source](https://security-hq.gutools.co.uk/documentation/snyk)
+- Internal tools should be behind Google Auth. (Network-layer restrictions may also be recommended based on the context)
+- All new 3rd party software / vendors used as part of a service should have approval via the [official process](https://spike.gnmremote.com/content/178/requesting-software-for-your-gnm)
+
+### Infrastructure
+- AWS infrastructure should *not* be deployed to the Developer Playground account
+- All new AWS infrastructure should be defined using [CDK](https://github.com/guardian/cdk), and legacy services should be migrated when possible
+- Infrastructure costs should be estimated and within existing budget (unless otherwise approved by Head/Director of Engineering)
+
+### Observability and support
+- Monitoring and alerting should exist to ensure the owner is notified when the core functionality of a service is unavailable or impaired
+- Application logs and (if necessary/applicable) telemetry data should be [shipped](https://github.com/guardian/deploy-tools-platform/tree/main/elk), so that service impairment can be debugged effectively for common problems (e.g. application errors, resource constraints)
+- For high priority applications, a runbook should be created to describe to other engineers how to debug and address service impairment incidents
+
+## High level maintenance expectations
+
+The broad expectations of ongoing maintenance of a production application are:
+1. Security vulnerabilities are addressed *as a priority*
+1. Dependencies are kept up to date, so that security patching requirements can be met efficiently
```

Comment:
Is it worth mentioning Dependabot and the internal Scala Steward lists here?

Reply:
We have a plan to overhaul the documentation on dependency management separate to this. I'll add a note to the card so that we come back here and add a link to it.